public/src/components/teacher/Results.tsx

```diff
@@ -116,7 +116,7 @@ export class Results extends React.Component<IResultsProp, IResultsState> {
     }
 
     private generateUserRepoLink(name: string, username: string): JSX.Element {
-        return <a href={this.props.courseURL + username + "-labs"}>{ name }</a>;
+        return <a href={this.props.courseURL + username + "-labs"} target="_blank">{ name }</a>;
```

Comment:
maybe this should use function or some other way to determine the user repo link. For example, maybe someone prefer to not add `-labs`?

Reply:
Will be fixed (constant fetched from the server) as a part of the PR with Student Page data fetching optimization.